saleor/graphql/order/mutations/draft_orders.py

```diff
@@ -146,16 +158,24 @@ def clean_input(cls, info, instance, data):
         billing_address = data.pop("billing_address", None)
         cleaned_input = super().clean_input(info, instance, data)
         lines = data.pop("lines", None)
-        channel = data.get("channel", None)
-        if "channel" in cleaned_input and channel is None:
+        channel_id = data.get("channel", None)
+        if "channel" in cleaned_input and channel_id is None:
             del cleaned_input["channel"]
-        cls.clean_channel_id(instance, channel)
+        cls.clean_channel_id(instance, channel_id)
+        voucher = cleaned_input.get("voucher", None)
+        if voucher:
+            channel = (
+                cleaned_input["channel"]
+                if "channel" in cleaned_input
+                else instance.channel
+            )
```

Comment:
```suggestion
            channel = cleaned_input.get("channel") or instance.channel
```

What do you think about this change? For me it's look more readable.